### ISSUE TITLE
Two entries in add-menu

### ISSUE THREAD

**Issue by spreeni151** [te-online/files_linkeditor#35]
The 1.1 version of Link Editor is working great, especially the 'open without confirmation' option is a big improvement. 
What might be confusing for non-technical users are the two entries for new links (.url and .webloc) in the 'add menu'. Is it possible to add a switch to the options menue to give admins the choice, which version is listed there (url _and/or_ webloc)?

**Comment by te-online** [comment]
Thanks for your feedback :-)
This sounds indeed like a good candidate for a configuration option. I can see how users can be confused, since no-one actually knows what .webloc or .URL means. I'll see what I can do, but in the meantime Pull Requests are always welcome!

**Comment by kusmi** [comment]
Yes, I would also like to choose for my users if I want .webloc or .URL (e.g. I want my users to use .webloc)

**Comment by fenvarien** [comment]
This configuration option would be very helpful. I need this app for education and my students are confused by the two options.

**Comment by Antoine-Derouin** [comment]
I also need this feature.

In the meantime I've put up this dirty css fix:
`.menuitem[data-templatename^="Link.webloc"] {display: none !important;}`

**Comment by spreeni151** [comment]
@Antoine-Derouin
In which file did you add this entry?
Thanks

**Comment by Antoine-Derouin** [comment]
@spreeni151 
The easiest way is to use the custom css plugin :
https://apps.nextcloud.com/apps/theming_customcss

**Comment by spreeni151** [comment]
Thank you very much Antoine! This is much easier and quicker to fix than I thought. I nearly forgot the custom css addon :-)

**Comment by it-1313** [comment]
Would it be possible to add checkboxes for URL and/or webloc in the admin config for this addon after all? The Link Editor app is great, but I think webloc is quite unknown and if you don't share Nextcloud with users who use a mac, I think webloc should be hidden to prevent confusion of users. Everbody should however understand the meaning of "URL".

I know there is the custom css "hack" but this would mean this app depends on another app (the cusom css app) for an important part of its functionality.

If you can share links to an example how such an admin page for the Link Editor could look like (e.g. from another addon), I can give it a try.

**Comment by te-online** [comment]
Hey @it-1313, thanks for posting here. Great that you are open to add this functionality 😊 

This guide should give you a few pointers: https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/settings.html

Let me know if I can help on any part of the way 👍 Preferably through a Draft Pull Request 🤓

**Comment by danth** [comment]
As a workaround for this you can delete one of the menu entries in `Linkeditor.service.js`:

https://github.com/te-online/files_linkeditor/blob/2738fe1c08cffd486203680471a36f86250b8d70/js/lib/Linkeditor.service.js#L90-L99

And perhaps simplify the display name of the remaining entry by removing the file format:

```js
displayName: window.t("files_linkeditor", "New link"),
```

Then rebuild `bundle.js` by using the command `npm run build`.

Modify the version number so that Nextcloud knows the code has changed (I just added `-hide-webloc` after the number):

https://github.com/te-online/files_linkeditor/blob/2738fe1c08cffd486203680471a36f86250b8d70/appinfo/info.xml#L20

You can now install the edited app, and the deleted menu entry should disappear. If the hidden file type is created by other means (e.g. upload), it will still be available to open and edit; only the button to create *new* links of that type is removed.

**Comment by rasos** [comment]
The CSS trick https://github.com/te-online/files_linkeditor/issues/35#issuecomment-780477147  does not work anymore with Nextcloud 29 to hide the add webloc. Our users are now confused what that means.  To have a config option to hide the webloc in the menu would be really great  I would like to avoid forking the app.

**Comment by te-online** [comment]
I'm sorry this is confusing for your users. I believe you may be able to find a new CSS rule to accomplish the same thing. Unfortunately, I don't really have the resources right now to add new feature to the app.
